### ISSUE TITLE
Normalize head date (server: Date, client: string)

### ISSUE THREAD

**Issue by MoOx** [phenomic/phenomic#397]
Following https://github.com/MoOx/phenomic/pull/395

gray-matter transform date field to Date. But on client side, we have raw json, so string. We should normalize that somewhere. Maybe after gray-matter call.

**Comment by thangngoc89** [comment]
Looks like this is a YAML feature see https://github.com/nodeca/js-yaml#supported-yaml-types.

**Comment by MoOx** [comment]
So maybe we should normalize in the emitted collection on the client side.

**Comment by thangngoc89** [comment]
I think normalize everything to string is better. Less code on client size. And less magic. I don't want user to ask us about what's going on

**Comment by thangngoc89** [comment]
Using `JSON.parse` and `JSON.stringify` should be the easiest (and probably fastest) way to fix this. But this might impact the loader's performance. Will open another issue about this.

**Comment by MoOx** [comment]
FYI https://github.com/yahoo/serialize-javascript/pull/16

I successfully used my PR but this works for collection only: pages are JSON. I AM SO FUCKIN STUPID.

Blocked by #434 for a full resolution, meanwhile I will force dates to be ISOString on node, but JS with dates is kind of ready.

**Comment by bloodyowl** [comment]
Going to be fixed with #925

**Comment by MoOx** [comment]
What is your plan for this?

**Comment by bloodyowl** [comment]
everything as a string, and we dont care about the format as long as it's parsable by `Date` in order to sort posts

**Comment by MoOx** [comment]
Not a problem with v1
